lib/tasks/cortex/core/media.rake

```diff
@@ -29,6 +31,7 @@ namespace :cortex do
                            },
                          metadata:
                            {
+                             naming_data: fieldTitle.id,
```

Comment:
Oh i thought it reduced the steps, as in just `[:naming_data]` as opposed to `[:naming_data][:title]`. Want me to switch it back?